tests/opts.py

```diff
@@ -10,6 +10,7 @@ class opts:
     '''Static class that encodes the global options.'''
     verbose = False
     tcpdump = False
+    tcpdump_dir = ''
     stop_on_failure = False
 
```

Comment:
Is this common practice? while it's convenient to keep these options on a static class I would assume they would be stored in either a `tests.config` , a json file file or something of that sort.

Reply:
A static class is a somewhat pythonesque way of organising global variables, yes. It has the advantage that you don't need the `global` keyword.